src/tests/unit/tests/DetailsView/components/__snapshots__/action-and-cancel-buttons-component.test.tsx.snap

```diff
@@ -2,19 +2,19 @@
 
 exports[`ActionAndCancelButtonsComponent render with primary button href == null 1`] = `
 <div
-  className="action-and-cancel-buttons-component"
+  className="actionAndCancelButtonsComponent"
```

Comment:
Is it intentional that it's camelCasing instead of kebab-casing? (Do we care?)

Reply:
Oh, sorry, I understand now; this isn't the "real" output, it's the output from the identity-obj-proxy magic

Reply:
Precisely! I think I may cover this in an engineering discussion. 😄